src/frontend/pages/chat/chat.js

```diff
@@ -405,7 +394,28 @@ export class Chat extends LitElement {
   }
 
   sendMessage(e) {
-    this.message = e.detail.message;
+    this.message = e.detail.message.replaceAll("\\\n", "\n");
```

Comment:
logica aggiunta alla funzione poco leggibile

si può dividere in più funzioni?
ha senso creare un servizio apposito o metterla in un servizio esistente?
alcune logiche sono duplicate nel repository?